src/test/ui/suggestions/const-pat-non-exaustive-let-new-var.rs

```diff
@@ -1,6 +1,6 @@
 fn main() {
     let A = 3;
-    //~^ ERROR refutable pattern in local binding: `i32::MIN..=1_i32` and
+    //~^ ERROR refutable pattern in local binding: `i32::MIN..=1` and
```

Comment:
This change is because we're now using `ConstInt` instead of `ty::Const` printing for these errors